src/burst_db/_land_usgs.py

```diff
@@ -0,0 +1,100 @@
+import fnmatch
+import zipfile
+from pathlib import Path
+
+import geopandas as gpd
+import pandas as pd
+import requests
+import unzip_http
+from shapely.geometry import MultiPolygon
+
+USGS_LAND_URL = "https://www.ngdc.noaa.gov/mgg/shorelines/data/gshhg/latest/gshhg-shp-2.3.7.zip"  # noqa
+GREENLAND_URL = "https://geodata.lib.utexas.edu/download/file/stanford-sd368wz2435-shapefile.zip"  # noqa
+
+
+def get_usgs_land(outpath=None):
+    f"""Get the USGS land data from {USGS_LAND_URL}."""
+    outpath = Path(outpath) if outpath else Path.cwd()
+    rzf = unzip_http.RemoteZipFile(USGS_LAND_URL)
+    # Level 1: Continental land masses and ocean islands, except Antarctica.
+    # Level 6: Antarctica based on grounding line boundary.
```

Comment:
Thanks for leaving the detailed description about the level 6 data, otherwise I would have overlooked it because I am not familiar with the GSHHG data ;)

I think this shape file would be okay for OPERA (the project does not cover Antarctica), but I would suggest to use another dataset whose boundary is based on coast line, not grounding lines. A lot of glaciologists are usually interested in not only grounded ice but also ice shelf (i.e. extension of grounded ice that floating on, and flows into the ocean)

The screenshot below is the difference between grounded ice (mint polygon) and the coastline
<img width="1851" alt="Screenshot 2023-06-15 at 15 15 52" src="https://github.com/opera-adt/burst_db/assets/27862199/e3b3bb8d-945e-428f-ba79-beb7dfe9f39c">

Reply:
Marking this comment as resolved to place this issue in lower priority and posting this issue in #10.